functions/_enhancd_filepath_abs.fish

```diff
@@ -3,13 +3,13 @@ function _enhancd_filepath_abs
     set -l cwd (string match --regex '^/.*/' "$PWD")
     set -l dir "$argv"
 
-    if [ -z "$dir" ] || [ -p /dev/stdin ]
+    if test -z "$dir"; or test -p /dev/stdin
```

Comment:
@gazorby We are already using fish `3.x` new features in some places, so might as well use `||`, rather than `; or` here.